lib/airrecord/table.rb

```diff
@@ -1,13 +1,18 @@
 module Airrecord
   class Table
     class << self
-      attr_accessor :base_key, :table_name, :api_key, :associations
```

Comment:
Dropped the `:associations` accessor while making this change, because v1 made it a noop.